src/ZipCode/Validator.php

```diff
@@ -4,6 +4,39 @@
 
 class Validator
 {
+    const EXAMPLES = [
```

Comment:
where do we need this for? i see only use case is to use in input placeholder

some of them allow 0, while others dont. so example need to have `0` where allowed, `1` where  not

also we can have more than one examples if a given regex supports multiple formats

Reply:
also what do you think we should merge two constants?
```php
const FORMATS = [
  'AT' => ['regex' => '[1-9]{1}[0-9]{3}', 'examples' => [1000, 1111]]
],
```

Reply:
Yep - that's the use case - to display examples on the front end however we want.

I think separate arrays are better - we may want to throw the example array out as a javascript object.  Keeping them separate will make this easier.